gitlab/v4/objects/__init__.py

```diff
@@ -3437,8 +3438,10 @@ def merge(
         data = {}
         if merge_commit_message:
             data["merge_commit_message"] = merge_commit_message
-        if should_remove_source_branch:
-            data["should_remove_source_branch"] = True
+        if should_remove_source_branch is not None:
+            data["should_remove_source_branch"] = bool(
+                strtobool(should_remove_source_branch)
```

Comment:
the only problem here is when `should_remove_source_branch` is already a bool. Then this will crash 🤔